rnd/autogpt_server/autogpt_server/server/server.py

```diff
@@ -18,12 +18,12 @@
 from autogpt_server.executor import ExecutionManager, ExecutionScheduler
 from autogpt_server.util.data import get_frontend_path
 from autogpt_server.util.process import AppProcess
-from autogpt_server.util.service import get_service_client
+from autogpt_server.util.service import get_service_client, expose  # type: ignore
+from autogpt_server.server.routes import websocket_endpoint as ws_impl
 from autogpt_server.util.settings import Settings
 
 
 class AgentServer(AppProcess):
```

Comment:
AgentServer is not yet a Pyro service (where you can add @expose method).

We need to replace AppProcess to AppService.